### PR TITLE
N52: add metalmancy opensauce interview

### PR DESCRIPTION
**Pull request by Vrixyz**
_Part of #1523_

### CODE REVIEW

File: content/news/052/index.md [hunk 1]
```diff
@@ -84,6 +85,21 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Other News
 
+### [Metalmancy @ OpenSauce][metalmancy_interview]
+
+![image/GIF description](metalmancy_interview.png)
+_Arcade cabinet close up_
+
+[Framework][framework_website] interviewed Carlos and Joanna on their games
+and arcade machines at [OpenSauce][opensauce_website].
```
Comment (Vrixyz) [inline content/news/052/index.md:94]:
```suggestion
[Framework][framework_website] interviewed Carlos and Joanna on their games
and arcade machines at [OpenSauce][opensauce_website]
at their stand for [Metalmancy][metalmancy_website].
```

Comment (Vrixyz) [reply content/news/052/index.md:94]: Not sure exactly, it just feels a bit weird that metalmancy is not introduced

File: content/news/052/index.md [hunk 2]
```diff
@@ -84,6 +85,21 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Other News
 
+### [Metalmancy @ OpenSauce][metalmancy_interview]
+
+![image/GIF description](metalmancy_interview.png)
+_Arcade cabinet close up_
+
+[Framework][framework_website] interviewed Carlos and Joanna on their games
```
Comment (cdsupina) [inline content/news/052/index.md:93]:
```suggestion
[Framework][framework_website] interviewed Carlo and Joanna on their games
```

File: content/news/052/index.md (hunk 1 above)
Comment (janhohenheim) [reply content/news/052/index.md]: Agreed. I wrote a little intro :)

**Review by janhohenheim — APPROVED** [review]

**Pull request merged**